app/helpers/clinicfinders_helper.rb

```diff
@@ -8,6 +8,8 @@ def parse_clinicfinder_field(clinic, field)
       "#{clinic[field].round(2)} miles"
     when :cost
       number_to_currency clinic[field], precision: 0
+    when :gestational_limit
+      clinic.gestational_limit.present? ? "Goes to #{(clinic.gestational_limit / 7).to_i}w" : 'Not specified'
```

Comment:
Possible rounding weirdness. I can see this being a bit odd, if the clinic takes patients up to, say, 4.5 weeks, if this displays '4 weeks', and the patient we're looking to place is at 4.25 weeks, would this misinform our CMs?

Reply:
I think you're right about this. I'll change to XwYd (e.g. 5w6d, the format we use elsewhere in the patient view).